Source/WTF/wtf/Int128.h

```diff
@@ -191,6 +194,8 @@ class alignas(16) UInt128Impl {
   // Returns the highest value for a 128-bit unsigned integer.
   friend constexpr UInt128Impl UInt128Max();
 
+  bool isHashTableDeletedValue() const { return lo_ == std::numeric_limits<uint64_t>::max() && hi_ == std::numeric_limits<uint64_t>::max(); }
```

Comment:
`UInt128Impl` should look like and behaves as if `uint128_t` because we are using `using UInt128 = UInt128Impl`. Thus, we should not have any methods.